fileServer.js

```diff
@@ -0,0 +1,67 @@
+ 'use strict';
+
+ var http = require('http');
+ var fs = require('fs');
+ var path = require('path');
+ var url = require('url');
+ var rootDir = __dirname + '/public';
+
+ const mimeType = {
+   'html': 'text/html',
+   'jpeg': 'image/jpeg',
+   'jpg': 'image/jpeg',
+   'png': 'image/png',
+   'js': 'text/javascript',
+   'css': 'text/css'
+ };
+
+ function send404Request(res) {
+   fs.readFile('public/404.html', 'UTF-8', function(err, html) {
```

Comment:
@Sjones352 - what happens if an error occurs reading the 404 file? The operations or dev person running your application server will want to know that.

👍 on the code style and patterns in this function - very clean, just needs error handling.